lib/datashake-ruby-sdk/review_index/v1/profiles.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Datashake
+  module ReviewIndex
+    class V1
+      class Profiles < BaseEndpoint
+        PATH = "profiles"
+
+        allow_parameters %i[
```

Comment:
should this be called `request_parameters` now?

Reply:
Yes, I think we can rename this as well. Done.